health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -0,0 +1,168 @@
+            {
+                label: t("HCM_AM_PROJECT_NAME"),
+                type: "text",
+            },
+            {
+                label: t("HCM_AM_BILL_ACTIONS"),
+                type: "custom",
+            },
+        ];
+
+        return baseColumns;
+    }, [, t]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)